src/workloads/tpch/denormalized/10/Q11.yml

```diff
@@ -0,0 +1,19 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-query"
+Description: |
+  Run TPC-H query 11 against the denormalized schema. Using an 'executionStats' explain causes each command to run its execution plan until no
```

Comment:
You might want to use this description for all other workloads.

Reply:
This information is available in the phases file for all those queries. For consistency, I updated query 11 to follow the same pattern as the others. The workload doesn't actually use execution stats at all, but the phase does.